test/testoffscreen.c

```diff
@@ -113,23 +113,23 @@ int main(int argc, char *argv[])
     if (!SDL_InitSubSystem(SDL_INIT_VIDEO)) {
         SDL_Log("Couldn't initialize the offscreen video driver: %s\n",
                 SDL_GetError());
-        return SDL_FALSE;
+        return 0;
```

Comment:
Are these not failure-returns? Why are you returning 0 instead of e.g. 1 ?

Reply:
> Are these not failure-returns? Why are you returning 0 instead of e.g. 1 ?

oops sry